cli/valet.php

```diff
@@ -289,6 +289,61 @@
     })->descriptions('Change the version of php used by valet', [
         'phpVersion' => 'The PHP version you want to use, e.g php@7.2',
     ]);
+
+    /**
+     * Tail log files.
+     */
+    $app->command('logs [-f|--follow] [-l|--lines=] [key]', function ($follow, $lines, $key = null) {
+        $defaultLogs = [
+            'php' => VALET_HOME_PATH.'/Log/php.log',
```

Comment:
`php` and `mysql` don't exist on mine; i'll take a look at figuring out why we're expecting this to exist and then either have us drop this line or add instructions.

Reply:
Gotcha – I'll await further instructions.

Anticipating `php` might be dropped, I've changed the example/guide text to `nginx`:

![replace-php-with-nginx](https://user-images.githubusercontent.com/1914481/56010322-94e9c700-5d26-11e9-8761-f85eb83bca56.png)


Cheers!

Reply:
Let's drop mysql and php for now. We can re-add them later if we figure out what their intended target is.

Reply:
Done! (138df82aa25073301ab962072f25b9c270101a87)